.gitlab/e2e_test_junit_upload.yml

```diff
@@ -10,7 +10,7 @@ e2e_test_junit_upload:
   dependencies:
     # We need to exhaustively list all the `new-e2e-…` jobs that produce junit reports here
     # to avoid downloading all the artifacts of all the jobs of all the previous stages.
-    - new-e2e-containers-main
+    - new-e2e-containers
```

Comment:
As a follow up, we could upload whenever there are auto runs, but it does require refactoring to depend only on tests that run within the same conditions. Working on a refactoring of this job in https://github.com/DataDog/datadog-agent/pull/22321